File: MojStomatolog/MojStomatolog.Models/Core/Payment.cs
namespace MojStomatolog.Models.Core
{
    public class Payment
    {
        public int Id { get; set; }
        public double Amount { get; set; }
        public DateTime PaymentDate { get; set; }
        public string PaymentNumber { get; set; } = string.Empty;
    }
}

File: MojStomatolog/UI/mojstomatolog_desktop/lib/models/payment.dart
import 'package:json_annotation/json_annotation.dart';

part 'payment.g.dart';

@JsonSerializable()
class Payment {
  int? id;
  int? paymentId;
  double? amount;
  DateTime? paymentDate;
  String? paymentNumber;

  Payment? payment;

  Payment();

  factory Payment.fromJson(Map<String, dynamic> json) =>
      _$PaymentFromJson(json);
  Map<String, dynamic> toJson() => _$PaymentToJson(this);
}
